### DISCUSSION TITLE
Entirely turn off the mouse

### DISCUSSION THREAD

**Discussion by dud225** [nesbox/TIC-80#2160] [General]
Hello

I'd like to completely turn off the mouse, even when not running a game e.g. on surf mode, as it has no use on devices missing a mouse. This question has been raised several times, but adding `poke(0x3FFB,0)` in https://github.com/nesbox/TIC-80/blob/main/config.lua didn't do the trick, the cursor is still appearing when launching TIC-80.
I've tried to call `SDL_ShowCursor(SDL_DISABLE)` in the [start function](https://github.com/nesbox/TIC-80/blob/31ea041978b63869589c1e49b6ee83e4e8ebdc54/src/system/sdl/main.c#L1703) to no avail.
Am I missing something?

**Comment by nesbox** [comment]
Also, try to comment this line https://github.com/nesbox/TIC-80/blob/main/src/system/sdl/main.c#L676
Sorry for the delay with the response

**Comment by dud225** [comment]
Here is what I tried:
```
diff --git a/src/system/sdl/main.c b/src/system/sdl/main.c
@@ -652,7 +667,8 @@ static void processMouse()
                 tic_point m = {(pt.x - rect.x) * TIC80_FULLWIDTH / rect.w, (pt.y - rect.y) * TIC80_FULLHEIGHT / rect.h};

                 if(m.x < 0 || m.y < 0 || m.x >= TIC80_FULLWIDTH || m.y >= TIC80_FULLHEIGHT)
-                    SDL_ShowCursor(SDL_ENABLE);
+                    //SDL_ShowCursor(SDL_ENABLE);
+                    SDL_ShowCursor(SDL_DISABLE);
                 else
                 {
                     SDL_ShowCursor(SDL_DISABLE);
@@ -1659,6 +1684,7 @@ static void emsGpuTick()
 static s32 start(s32 argc, char **argv, const char* folder)
 {
     SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO | SDL_INIT_JOYSTICK);
+    SDL_ShowCursor(SDL_DISABLE);
     platform.studio = studio_create(argc, argv, TIC80_SAMPLERATE, SCREEN_FORMAT, folder);
 
     SCOPE(studio_delete(platform.studio))
```

but the cursor is still showing up at the middle of the screen.

**Reply by nesbox** [reply]
Also try to disable `blitCursor()` function in `studio.c`

**Reply by dud225** [reply]
Thanks it worked.
Actually it isn't necessary to add `SDL_ShowCursor(SDL_DISABLE)`, just commenting `blitCursor()` is enough.

**Comment by dud225** [comment]
Commenting `blitCursor()` function in studio.c does the trick
